### DISCUSSION TITLE
Miss detections on Small Size text with same font

### DISCUSSION THREAD

**Discussion by sharoseali** [PaddlePaddle/PaddleOCR#13164] [Q&A]
Hi, thanks for your great work. I am applying PPOCR on a pdf page having architectural drawings. The problem is some text is not been detected, however same font text in the same page is also detected at a smaller distance.  For example, in the below image some of the C3 and C4 text is detected but some are missed. Can anyone help with this? How can I resolve this issue?
The detector I am using is **DB++**

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/19869426/04c993a4-b844-491b-9593-5323fac49eeb)

Thanks

**Comment by UserWangZz** [comment]
1. Try the ppocr-v4 version of the model
2. During inference, try to change det_limit_side_len, det_db_box_thresh
det_db_score_mode and other parameters


You can refer to the following code：
https://github.com/PaddlePaddle/PaddleOCR/blob/433677182f108c1be413ee8a92815bc13b205737/tools/infer/utility.py#L53-L68

**Reply by sharoseali** [reply]
Thanks for your reply. I have updated the my ppocr python package via pip to 2.6.1. The downloaded model's folder starts from **ch** like  `ch_PP-OCRv4_server_det`. How can I download the English language model?   Among all the models shown below, which detection and recognition models are the the best in performance while inference.

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/19869426/5e389a04-2e90-44e3-baf3-667a8d1a918a)

thanks
